### DISCUSSION TITLE
Latest VDH "URL COPY" copies masterManifest URL

### DISCUSSION THREAD

**Discussion by sabotendaaaa** [aclap-dev/video-downloadhelper#80] [Help and Q&A]
### Your question

Old version VDH: When I clicked URL COPY button, VDH copied mediaManifest URL.
Latest version(8.2.0.20) VDH: When I click URL COPY button, VDH copies masterManifest URL.

for example;
when I want to get 1280x720 video's m3u8 URL, I choose 1280x720 and click URL COPY,
but VDH copies master m3u8, not 1280x720 m3u8.
![vdh](https://github.com/aclap-dev/video-downloadhelper/assets/123542342/88ff7281-cf69-42fb-8e0d-f3128d253f92)

**Comment by paulrouget** [comment]
Correct, we copy the master manifest. Copying the media manifest is usually not the right approach as you might be missing the audio. But then you lose the ability to choose the video quality.

What if we copy both the video and audio manifest ? You would end up with 2 urls though.

**Reply by sabotendaaaa** [reply]
paulrouget san,

Thanks your reply.

So, VDH changes URL copy's action from copy media-manifest to copy master-manifest
and this change is not a bug/mistake but a right/designed update, right?

Umm, it was useful for me to copy media-manifest directly & 1 click, but I understand your explanation.
